src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -11,6 +11,9 @@
 	Added passive PII Scanner (which currently looks for Credit Card # patterns).<br>
 	Change Image Location and Privacy Scanner to scan hidden images.<br>
 	Added blank link target scanner.<br>
+	Change Image Location and Privacy Scanner to escape HTML inside of embedded image comments.<br>
```

Comment:
The version needs to be bumped and old changes removed.

Reply:
I'm not sure what you mean by this.  The version of the xml file at the top?  And what is meant by old changes?

Reply:
Yes, the version should be 20. I'm referring to the previous changes (above this line, from "Minor code.." to "Added blank...").

Reply:
Committed.  I think that's it for your required updates.